.gitignore

```diff
@@ -0,0 +1,40 @@
+# # I'm not sure if this can safely be added?
+## MkDocs build directory
+#site/
```

Comment:
This will depend on how these docs are built and released, I'm assuming :thinking: